vyper/cli/vyper_serve.py

```diff
@@ -66,7 +66,7 @@ def do_GET(self):
             self.send_response(200)
             self.send_cors_all()
             self.end_headers()
-            self.wfile.write('Vyper Compiler. Version: {} \n'.format(vyper.__version__).encode())
+            self.wfile.write('Vyper Compiler. Version: {(vyper.__version__).encode()}\n')
```

Comment:
Missing the format string delimter
```suggestion
            self.wfile.write(f'Vyper Compiler. Version: {(vyper.__version__).encode()}\n')
```

Reply:
Wait, should that even have curly braces at all? The `encode` should mean that there's no variable being called. Maybe it should be something like:
```suggestion
            self.wfile.write('Vyper Compiler. Version: ' + (vyper.__version__).encode() + '\n')
```

Reply:
Sure, that looks fine

Reply:
I ran tests on my suggestion, and I can say that it does compile.